### PR TITLE
Update custom screen reset data

### PR DESCRIPTION
**Pull request by ektaghag-eaton**
Fixes # .

#### Changes proposed in this Pull Request:

-
-
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: login-workflow/docs/registration-workflow.md [hunk 1]
```diff
@@ -113,5 +113,3 @@ The RouteConfig is an object that specifies the paths you are using for the rout
 | screenData           | `ScreenData`                           | An object of data of all screens available in the Registration Workflow Context. Check [ScreenData](#screendata) for more details. |         |
 | totalScreens         | `number`                               | The total number of screens in the registration workflow.                                                                          |         |
 | updateScreenData     | `(data: IndividualScreenData) => void` | Updates collected data/inputs throughout the Registration Workflow.                                                                |         |
-
-> To reset the workflow data when the user clicks on the close icon from the header, the custom screen should call `resetScreenData()` function.
```
Comment (JeffGreiner-eaton) [inline login-workflow/docs/registration-workflow.md:117]: Isn't currently used? why are we removing this section

Comment (JeffGreiner-eaton) [reply login-workflow/docs/registration-workflow.md:117]: never mind I see the api table entry, should the resetScreenData description reflect both the close button and or clicking back all the way out of registration workflow clears collected data or something like that

Comment (surajeaton) [reply login-workflow/docs/registration-workflow.md:117]: `resetScreenData` should be called on all the screen's close buttons in the header.
But, only the first screen's back button in the workflow should be hooked with `resetScreenData`, while other screens should persist in the input screen data, navigating back.

File: login-workflow/docs/customization.md [hunk 2]
```diff
@@ -62,26 +61,51 @@ Your custom implementation, removing the EulaScreen and VerifyCodeScreen, and ad
...
 import { SuccessScreenBase } from '@brightlayer-ui/react-native-auth-workflow';
 
 const MyCustomSuccessScreen = () => {
-  return (
-    <SuccessScreenBase 
-        messageTitle={'Congratulations!'}
-        message={'You have been registered successfully'}
-    />
-  );
+    return <SuccessScreenBase messageTitle={'Congratulations!'} message={'You have been registered successfully'} />;
```
Comment (surajeaton) [inline login-workflow/docs/customization.md:102]: This is incorrect, after refactoring the success screen base to have `EmptyStateProps`.
Do we want to cover it in this PR, or a separate one?

Comment (ektaghag-eaton) [reply login-workflow/docs/customization.md:102]: I will update this PR. Thanks for pointing out.

**Review by JeffGreiner-eaton — APPROVED** [review]

**Pull request merged**